src/content/quickstart/verify_with_discovery.md

```diff
@@ -10,6 +11,19 @@ kubectl -n default expose deployment nginx --port=8080
 subctl export service --namespace default nginx
 ```
 
+##### Deploy Headless Service
```

Comment:
Do we have section that explains about headless service in relation to Submariner ?
I would suggest to refer to: 
https://kubernetes.io/docs/concepts/services-networking/service/#headless-services

Reply:
I don't think that there is anything unique about headless services and Submariner. We can perhaps link to this URL somewhere, but there is nothing specific to Submariner we should document AFAIK (similarly we don't have any documentation of what is a ClusterIP service, there are standard k8s definitions we follow/implement).